src/logic/hooks/useExecutionStatus.ts

```diff
@@ -1,22 +1,26 @@
-import { EstimationStatus } from 'src/logic/hooks/useEstimateTransactionGas'
+import { DEFAULT_GAS, EstimationStatus } from 'src/logic/hooks/useEstimateTransactionGas'
 import { useEffect, useState } from 'react'
 import useAsync from 'src/logic/hooks/useAsync'
+import { DEFAULT_GAS_LIMIT } from 'src/logic/hooks/useEstimateGasLimit'
 
 export const useExecutionStatus = (
   checkTxExecution: () => Promise<boolean>,
   isExecution: boolean,
   txData: string,
   gasLimit: string | undefined,
+  gasPrice: string,
+  gasMaxPrioFee: string,
 ): EstimationStatus => {
   const [executionStatus, setExecutionState] = useState<EstimationStatus>(EstimationStatus.LOADING)
 
   const [status, error, loading] = useAsync(async () => {
     if (!isExecution || !txData) return EstimationStatus.SUCCESS
-    if (!gasLimit) return EstimationStatus.LOADING
+    if (!gasLimit || gasLimit === DEFAULT_GAS_LIMIT || gasPrice === DEFAULT_GAS || gasMaxPrioFee === DEFAULT_GAS)
```

Comment:
Can we put this condition in a variable?

Reply:
Good idea!

Reply:
done